alpine/content.md

```diff
@@ -0,0 +1,30 @@
+# What is Alpine Linux?
+
+[Alpine Linux](http://alpinelinux.org/) is a Linux distribution built around [musl libc](http://www.musl-libc.org/) and [BusyBox](http://www.busybox.net/). The image is only 5 MB in size and has access to a [package repository](http://forum.alpinelinux.org/packages) that is much more complete than other BusyBox based images. This makes Alpine Linux a great image base for utilities and even production applications. [Read more about Alpine Linux here](https://www.alpinelinux.org/about/) and you can see how their mantra fits in right at home with Docker images.
+
```

Comment:
Can you add `%%LOGO%%` on a line by itself here so we can show off that perty logo? :+1: